webmails/rainloop/config/nginx-rainloop.conf

```diff
@@ -12,13 +12,16 @@ server {
 
     index index.php;
 
+    # set maximum body size to configured limit
+    client_max_body_size {{ MESSAGE_SIZE_LIMIT|int + 8388608 }};
```

Comment:
you just reintroduced the bug i was fixing in [#967ebad](https://github.com/Mailu/Mailu/pull/2187/commits/967ebad94d9103309a60f2d5c3de6cb213974625)

Reply:
Actually, I am following @nextgens comment https://github.com/Mailu/Mailu/pull/2187#discussion_r795542809 not to support setting message size limit 0.  I do not believe that it should be allowed.

Reply:
@audioscavenger I discovered something that is probably also applicable to #2187.  If I recreate the webmail container from my repo, tag it mail/webmail:1.9 in my local docker host and run docker-compose up -d then the change to webmail is not effective and I cannot upload a 9MB attachment.  If I docker-compose restart webmail, then the same attachment will work.  This imply that if I start all containers at the same time, things don't work.  If I just restart webmail, the change in /etc/nginx/nginx.conf (client max body size set to 0 disable) seems to take effect.

Reply:
no idea what you mean, sorry. are you okay with limit=0 or not is the question

Reply:
I said ealier that I agreed that @nextgens that size 0 is not desirable. As for what I am talking about: when I test your changes or mine I update the webmail container and then run mailu with docker-compose. When I do this it fails until I use docker-compose restart for webmail. Is it not how you test and do you not get the same results?

Reply:
Ok, it took a while but I finally figured out what was going on.  The issue is that the Dockerfile for webmail starts the fastcgi process manager and then updates the 3 *.ini files that it uses.  This cause those configuration changes to only take place when restarting the container.  I have modified the Dockerfile to not start the fastcgi process manager and update the start.py to start it after making the configuration changes using jinja.  Now the MESSAGE_SIZE_LIMIT configured in the mailu.env file are respected in both nginx and php-fpm7 on the webmail container.  I have of course tested those changes.  This can be reviewed and is ready to pull back into master.